src/select/select.tsx

```diff
@@ -69,13 +69,13 @@ class Select<T, O extends Option<T> = Option<T>> extends React.Component<Props<T
             <div
                 ref={this.setRootNode}
                 role="button"
-                className={`select-component ${this.props.className || ''}`}
+                className={`select-component ${this.props.className || ''} ${this.state.isExpanded ? 'selected-active':''}`}
                 data-qa={`select-component-${this.props.id}`}
                 onKeyDown={this.onKeyPressed}
             >
                 {this.props.searchable ? this.renderSearchableTrigger() : this.renderStandardTrigger()}
                 {this.state.isExpanded && Boolean(options.length) && (
-                    <div className="select-options-wrapper" data-qa={`select-options-wrapper-${this.props.id}`}>
+                    <div className={`select-options-wrapper ${this.state.isExpanded ? 'selected-active':''} `} data-qa={`select-options-wrapper-${this.props.id}`}>
```

Comment:
`this.state.isExpanded` is always true when the element is rendered. you can just add the class.